### ISSUE TITLE
Internal Server Error - 500 - Graph Beta auditLogQuery

### ISSUE THREAD

**Issue by KaveSys** [microsoftgraph/microsoft-graph-explorer-v4#3482]
**Describe the bug**
I receive (both from Graph Explorer and Powershell 7) the error: Internal Server Error - 500 when run a POST to create an auditLogQuery search

**To Reproduce**
1. Go to https://developer.microsoft.com/en-us/graph/graph-explorer
2. Select 'POST' and beta
3. Search https://graph.microsoft.com/beta/security/auditLog/queries
4. Put Request body:
{
   "displayName": "Audit Search 06-Dec-2024 09:51",
   "filterStartDateTime": "2024-11-06T00:00:00.000Z",
   "filterEndDateTime": "2024-12-06T00:00:00.000Z",
   "operationFilters": [
    "filepreviewed",
    "filedownloaded",
    "fileaccessed"
  ]
 }

**Expected behavior**
Create the auditLogQuery search

**Screenshots**
![image](https://github.com/user-attachments/assets/e4f39e09-3749-4699-ba7c-a4332beca9c2)

**Desktop (please complete the following information):**
 - OS: Windows 11 - Powershell 7.4.6
 - Browser Edge 131.0.2903.70

**Comment by ElinorW** [comment]
Hi @KaveSys,
This is likely an issue with the API. Please post this on the[ Microsoft Graph Q&A forum](https://learn.microsoft.com/en-us/answers/tags/161/ms-graph), where the appropriate team can reach out and help

**Comment by KaveSys** [comment]
@ElinorW sorry but is not clear why I need to open another forum post if GitHub should already be used to solve problems and precisely for this purpose

**Comment by ElinorW** [comment]
> @ElinorW sorry but is not clear why I need to open another forum post if GitHub should already be used to solve problems and precisely for this purpose

Hey @KaveSys , 
this is because the Graph Explorer team only handles issues specific to the Graph Explorer web app. Other teams handle the Graph API's and when you would like to report an issue related to how the API is working, it's advisable to post it there

**Comment by ElinorW** [comment]
> @ElinorW you read carefully the issue? "both from Graph Explorer and Powershell 7", so the issue is related to API, not only the web app :)

Yeah, this is an issue with the API. Several teams work on Graph API. When you post the issue on the Graph Q&A forum, the appropriate team will be able to help
